compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/transform/CreateViewTransform.java

```diff
@@ -40,7 +40,8 @@
  */
 public final class CreateViewTransform<I, O> implements Transform<WindowedValue<KV<?, I>>, WindowedValue<O>> {
   private final ViewFn<Materializations.MultimapView<Void, ?>, O> viewFn;
-  private final Map<BoundedWindow, List<I>> windowListMap;
+
+  private final transient Map<BoundedWindow, List<I>> windowListMap;
```

Comment:
Please just remove the transient keyword and leave it untouched. This is rather a complex issue, so let's not resolve it.